### PR TITLE
Allow filtering passagehouraggregation

### PR DESCRIPTION
**Pull request by snelis**

### CODE REVIEW

File: api/Makefile [hunk 1]
```diff
@@ -19,3 +28,15 @@ requirements: pip-tools             ## Upgrade requirements (in requirements.in)
 	pip-compile --upgrade --output-file requirements_dev.txt requirements_dev.in
 
 upgrade: requirements install       ## Run 'requirements' and 'install' targets
+
+build: export BUILD_DATE=$(build_date)
```
Comment (snelis) [inline api/Makefile:32]: remove

File: api/deploy/test/test [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```
Comment (snelis) [inline api/deploy/test/test:1]: delete

File: api/src/passage/views.py [hunk 3]
```diff
@@ -16,6 +17,11 @@
 from . import models, serializers
 
 
+class ExportFilter(FilterSet):
```
Comment (snelis) [inline api/src/passage/views.py:20]: weg

File: api/src/passage/tests/test_api.py [hunk 4]
```diff
@@ -255,3 +255,29 @@ def test_passage_export(self):
...
+            hour=1,
+            _quantity=100,
+        )
+        url = reverse('v0:passage-export')
+        response = self.client.get(url, dict(year=2019, week=12))
+        self.assertEqual(response.status_code, 200)
+        lines = [x for x in response.streaming_content]
+        assert len(lines) == 0
+
+        response = self.client.get(url, dict(year=2019))
```
Comment (snelis) [inline api/src/passage/tests/test_api.py:278]: add week=11 test

**Review by kramer65 — APPROVED** [review]

**Pull request merged**